src/main/java/de/maxhenkel/pipez/Main.java

```diff
@@ -80,4 +81,8 @@ public void clientSetup(FMLClientSetupEvent event) {
         Containers.clientSetup();
     }
 
+    public static ResourceLocation rl(String location) {
+        return new ResourceLocation(MODID, location);
+    }
+
```

Comment:
Please leave the resource locations like they are. I do it like this in every of my mods. I want to keep it consistent with my other mods.

Reply:
Sure thing.